tree/ntuple/v7/doc/BinaryFormatSpecification.md

```diff
@@ -1059,6 +1059,17 @@ The limits refer to a single RNTuple and do not consider combinations/joins such
 | Maximum string length (meta-data)              | 4GB                          | String encoding                                        |
 | Maximum RBlob size                             | 128 PiB                      | 1GiB / 8B * 1GiB (with maxKeySize=1GiB, offsetSize=8B) |
 
+## Naming specification
+
+The allowed character set for the strings representing the name of an RNTuple, as well as the name of a field, is
+restricted to printable Unicode characters encoded as UTF-8, with the following exceptions:
+
+* Full stop (U+002E, `.`)
+* Space (U+0020, ` `)
```

Comment:
I'd say all white spaces (space, tab, small space etc.)

Reply:
Ok, let me check the full list and come back to this, I'd like to be as specific as possible.